drivers/clone.c

```diff
@@ -136,11 +142,21 @@ static int parse_args(size_t numargs, char **arg)
 		if (!strcasecmp(arg[1], "ups.status")) {
 			snprintf(ups.status, sizeof(ups.status), "%s", arg[2]);
 
-			online = strstr(ups.status, "OL") ? 1 : 0;
+			/* Status is published later in upsdrv_updateinfo() */
+			return 1;
+		}
+
+		if (ups.load.status && !strcasecmp(arg[1], ups.load.status)) {
+			if (!strcasecmp(arg[2], "off") || !strcasecmp(arg[2], "no")) {
+				outlet = 0;
```

Comment:
Sets internal state from data we have, for later updating from directly in `upsdrv_updateinfo()`, instead of then needing to `dstate_getinfo` it again otherwhere to the same effect.